photos/photos.py

```diff
@@ -155,56 +184,121 @@ def watermark_photo(image, watermark_text, watermark_image, watermark_image_size
     return image
 
 
-def resize_worker(orig, resized, spec, wm, wm_text, wm_img, wm_img_size):
+def rotate_image(img, exif_dict):
+
+    if "exif" in img.info and piexif.ImageIFD.Orientation in exif_dict["0th"]:
+        orientation = exif_dict["0th"].pop(piexif.ImageIFD.Orientation)
+        if orientation == 2:
+            img = img.transpose(Image.FLIP_LEFT_RIGHT)
+        elif orientation == 3:
+            img = img.rotate(180)
```

Comment:
we could (should?) use transpose() with Image.ROTATION_180 etc  instead of rotate(). 

reason being I had an issue where rotate did no longer rotate the dimensions of the image and I got black bars+cropping as a result (maybe i updated Pillow too far?)